site/content/docs/02-template-syntax.md

```diff
@@ -1639,7 +1639,7 @@ The `<svelte:element>` element lets you render an element of a dynamically speci
 
 The only supported binding is `bind:this`, since the element type specific bindings that Svelte does at build time (e.g. `bind:value` for input elements) does not work with a dynamic tag type.
 
-If `this` has a nullish value, a warning will be logged in development mode.
+If `this` has void tag name (e.g. `br`) and has child element(s), runtime throw error and nothing to render in development mode.
```

Comment:
I felt that we should mention this validation, so I added the document.
And original sentence is incorrect (now it doesn't show warning) so I removed.